sdk/healthinsights/health-insights-radiologyinsights-rest/review/health-insights-radiologyinsights.api.md

```diff
@@ -42,6 +46,15 @@ export type AzureHealthInsightsClient = Client & {
     path: Routes;
 };
 
+// @public (undocumented)
+export function buildMultiCollection(items: string[], parameterName: string): string;
+
+// @public
+export type ClinicalDocumentType = string | "consultation" | "dischargeSummary" | "historyAndPhysical" | "radiologyReport" | "procedure" | "progress" | "laboratory" | "pathologyReport";
```

Comment:
Is this supposed to be an extensible enum? I think the type ends up being just `string` and I don't know if adding the specific values in this way help out with intellisense...

Reply:
It is a fixed list for radiology insights but the same ClinicalDocumentType might be used in other healthinsights projects, hence the generic extension

Reply:
I see, unfortunately what ends up happening is that TypeScript resolves this to a string and customers who use this type do not get any assistance. What we do in those cases is just make the type a `string` and provide an enum that is compatible. Then document the relationship.

Check out this typescript playground that shows what I mean: [playground](https://www.typescriptlang.org/play/?#code/KYDwDg9gTgLgBDAnmYcDCAbAlgOywYwEMMARCfAVwFtgcYAVZVAXjgGcYpcBzOAHzgAifBBxsKGGIRhZRg-kIAmWNvgAWhKN2ABlalU2J5AwWpUxoiAII5FABTWI2BYsaFRCyiBgjdEAJWBIWDdBMChyYEUKKGBQ8N9YtjZQjEIAI2hpS3jpNW9fAKDoGEEAbgAoCpExeCpERhQALnRsPCJScmpaBiY4VkFyoA)

@joheredi Do RLCs support extensible enums? 

If not, since this is an input type (I think, because I also see an output type below), maybe we just make it the enum values until we know of other use cases. We can always expand it by adding `string` later (you can widen input types, but not output types)